app/templates/pages/auth.jinja

```diff
@@ -24,4 +30,64 @@
         </div>
     </div>
 </div>
+
+
+
+<script>
+    // Get clientId from local storage
+    let clientId = localStorage.getItem('clientId');
+
+    if (clientId) {
+        // If clientId is set, create login request
+        const endpoint = `{{ url_for("auth.login") }}?client_id=${clientId}`
+        $.get(endpoint,
+            function (data, status) {
+                document.getElementById('start-auth').addEventListener('click', async () => {
```

Comment:
Would stick to one of vanilla JS or jQuery for selectors

Reply:
I've changed this for a jQuery selector